trunk/SUAVE/Methods/Propulsion/propeller_design.py

```diff
@@ -197,16 +207,37 @@ def propeller_design(prop_attributes):
     
     MCA = c/4. - c[0]/4.
     
-    
-    Power = Pc*rho*(V**3)*np.pi*(R**2)/2
-    Cp    = Power/(rho*(n**3)*(D**5))
-
-    prop_attributes.twist_distribution = beta
-    prop_attributes.chord_distribution = c
-    prop_attributes.Cp                 = Cp
-    prop_attributes.mid_chord_aligment = MCA
-    
-    #These are used to check, the values here were used to verify against
-    #AIAA 89-2048 for their propeller
-    
-    return prop_attributes
\ No newline at end of file
+    Thrust = Tc*rho*(V**2)*np.pi*(R**2)/2
+    Power  = Pc*rho*(V**3)*np.pi*(R**2)/2
+    Cp     = Power/(rho*(n**3)*(D**5))
+    
+    # compute max thickness distribution using NACA 4 series eqn
+    t_max          = np.zeros(20)
+    for idx in range(20):
+        c_blade    = np.linspace(0,c[idx],20)          # local chord  
+        t          = (5*c_blade)*(0.2969*np.sqrt(c_blade) - 0.1260*c_blade - 0.3516*(c_blade**2) + 0.2843*(c_blade**3) - 0.1015*(c_blade**4)) # local thickness distribution
+        t_max[idx] = np.max(t)                       
+ 
+    # Nondimensional thrust
+    if prop.design_power == None: 
+        prop.design_power = Power[0]        
+    elif prop.design_thrust == None: 
+        prop.design_thrust = Thrust[0]      
+    
+    # approximate thickness to chord ratio 
+    t_c               = t_max/c
+    t_c_at_70_percent = t_c[int(N*0.7)]
+    
+    prop.design_torque              = Power[0]/omega
+    prop.max_thickness_distribution = t_max
+    prop.twist_distribution         = beta
+    prop.chord_distribution         = c
+    prop.Cp                         = Cp
```

Comment:
Write out this variable

Reply:
Variable spelled out

Reply:
Variable written out: Cp changed to power coefficient